### PR TITLE
Hfp 3539 bq add exit button

### PR DESCRIPTION
**Pull request by ravimajithia**

### CODE REVIEW

File: branchingQuestion.js [hunk 1]
```diff
@@ -43,12 +43,66 @@ H5P.BranchingQuestion = (function () {
...
+      close.setAttribute('title', H5P.t('close'));
+
+      close.addEventListener('keyup', function (event) {
+        // Add support for space and enter
+        if (event.code === 'Enter' || event.code === 'Space') {
+          closeDialog();
+        }
+      });
+
+      close.onclick = function (e) {
```
Comment (fnoks) [inline branchingQuestion.js]: Please be consistent. I.e: use addEventListener('close', <...>) instead of onclick.

File: branchingQuestion.js [hunk 2]
```diff
@@ -43,12 +43,66 @@ H5P.BranchingQuestion = (function () {
...
+      return close;
+    };
+
+    /**
+     * Remove BQ and navigate back to its previous position
+     */
+    const closeDialog = function () {
+      const overlay = self.parent.libraryScreen.overlay;
+      if (overlay) {
+        // TODO: When does this code every run?!
```
Comment (fnoks) [inline branchingQuestion.js]: This TODO should be resolved. I.e: I don't understand why this code was added if you don't know when it is executed?

File: branchingQuestion.js [hunk 3]
```diff
@@ -43,12 +43,66 @@ H5P.BranchingQuestion = (function () {
...
       return wrapper;
     };
 
+    /**
+     * Create close element and register events
+     *
+     * @return {Element} close
+     */
+    const createCloseElement = function () {
+      const close = document.createElement('div');
```
Comment (fnoks) [inline branchingQuestion.js]: I believe we should try to use "button" instead. then we don't need to add special handling of the key events.

File: branchingQuestion.js (hunk 2 above)
Comment (ravimajithia) [reply branchingQuestion.js]: Already improved, just forgot to remove the message.

**Pull request merged**